### DISCUSSION TITLE
org.openrewrite.java.spring.boot2.MoveAutoConfigurationToImportsFile running on multi-module Gradle project

### DISCUSSION THREAD

**Discussion by erwinkantner79** [openrewrite/rewrite#3816] [Q&A] [answered]
I am struggling running the above rewrite recipe in a Gradle project with multiple modules. It runs throw without errors, but no changes are made. If I run the same recipe in a single project, the changes are made.
When I try other recipes they are executed on the submodules in the project with several modules. 
For me it seems that the above mentioned recipe is not executed with submodules. 
Has anyone had the same experience or any ideas what I can try?

**Comment by timtebeek** [comment, marked as answer]
Hi @erwinkantner79 ; Sorry to hear about those issues (and to see this was missed earlier). If other recipes are working then this is perhaps best logged as an issue over on rewrite-spring: https://github.com/openrewrite/rewrite-spring/issues/new/choose
